pydatastructs/trees/tests/test_suffixtree.py

```diff
@@ -0,0 +1,7 @@
+from pydatastructs import SuffixTree
+
+def test_suffixtree():
+    s = SuffixTree("HelloworldHe")
+    assert s.find("Hel") == 0
+    assert s.find_all("He") == {0, 10}
+    assert s.find("Win") == -1
```

Comment:
Please provide references for the tests for verifying the results. You may use examples from university lecture notes, wikipedia, books, etc.

Reply:
Still awaited.